packages/v4/patternfly-docs/content/design-guidelines/components/skeleton/skeleton.md

```diff
@@ -5,14 +5,22 @@ section: components
 
 ## Usage 
 Skeleton loading should be progressive –– it should load static text first, followed by variable elements of the UI.
+A skeleton is a type of loading state that allows you to expose content incrementally, once the structure of the page has been loaded in. A skeleton should match the exact structure of the element you’re loading in and can be used in components on top of the page, such as in a drop down menu, or a card.
 
 ### When to use a skeleton vs. a loading spinner
+- The structure of a page has loaded in but the content has not
+- When some content of a page has loaded, but not all. For example, in a dashboard, you may have some loaded cards, while others are still loading. 
+- You want to show loading content inside of a card, table, data list or more.
 
 **Show a skeleton when:**
 - You know what the populated data is going to look like (even if it results in an empty state).
 
+**Do not show a skeleton when:**
+- When you don’t know what the structure of the component is going to look like
+
 **Show a [spinner](/components/spinner/react) when:**
-- You do not know what the populated data may look like. For example, if you are loading a form (as they are not all structured the same).
+- When you have multiple elements on the page that are loading at different speeds, use a spinner once the structure of at least one element is loaded on the page
+- You are attempting to load in a component that sits on top of a page with an expected structure
 - When it is likely it will fail or show an empty state while waiting for actions to complete.
```

Comment:
same here with the "when" - your bullets should continue the header in a sentence that flows. So "Show a spinner when............ [bullet] it is likely it will ...."